adapter_test.go

```diff
@@ -15,14 +15,14 @@
 package gormadapter
 
 import (
+	"github.com/jackc/pgconn"
```

Comment:
Format the code, this dependency should move below.

Reply:
fixed